modal/_functions.py

```diff
@@ -1006,10 +1011,6 @@ async def _load(param_bound_func: _Function, resolver: Resolver, existing_object
 
         fun: _Function = _Function._from_loader(_load, "Function(parametrized)", hydrate_lazily=True)
 
-        if can_use_parent and parent.is_hydrated:
```

Comment:
I don't remember if there were cases where this was a crucial code path - I hope not!

Reply:
github reactions need to add the "see no evil" monkey